expr/log.go

```diff
@@ -17,43 +17,99 @@ package expr
 import (
 	"encoding/binary"
 
+	"github.com/google/nftables/binaryutil"
 	"github.com/mdlayher/netlink"
 	"golang.org/x/sys/unix"
 )
 
+type LogLevel uint32
+
+const (
+	// See https://git.netfilter.org/nftables/tree/include/linux/netfilter/nf_tables.h#n1226
```

Comment:
Here and below, can you make these links that won’t break when the file changes? E.g. https://git.netfilter.org/nftables/tree/src/datatype.c?id=488356b895024d0944b20feb1f930558726e0877#n1162